src/Umbraco.Web/Trees/ContentTreeControllerBase.cs

```diff
@@ -320,10 +320,10 @@ private TreeNodeCollection GetTreeNodesInternal(string id, FormDataCollection qu
 
             //before we get the children we need to see if this is a container node
 
-            //test if the parent is a listview / container
+            //test if the parent is a listview / container -> the result should be an even number

```

Comment:
I guess the comment can be updated since we aren't dealing with the INT, even number result now